src/pages/ApplicationDetails.tsx

```diff
@@ -26,7 +26,7 @@ export default function ApplicationDetailsPage() {
         const packageData = await getPackage(id);
         const versionData = await getReleases(id);
         setApplicationInformation({
-          package: packageData,
+          package: packageData as Package,
```

Comment:
Assure object has correct type before using

Reply:
Added if check before set state as packageData is always Package or null